.buildkite/commands/distribute_prototype.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -eu
+
+"$(dirname "${BASH_SOURCE[0]}")/setup_shared.sh"
+
+echo "--- :arrow_down: Downloading Prototype Build"
+buildkite-agent artifact download ".build/artifacts/*.ipa" . --step prototype_build
+buildkite-agent artifact download ".build/artifacts/*.app.dSYM.zip" . --step prototype_build
+
+echo "--- :firebase: Distributing Prototype Build"
+bundle exec fastlane distribute_prototype
```

Comment:
Most, if not all, our previous implementations use the verb "upload". For some reason, this time around the verb "distribute" felt more appropriate.

- Pro: The automation uploads the build and makes it immediately available. Using upload would tell only half of the picture.
- Con: The fact that the distribution is automatic is not really something that we control from this side of the code, so maybe the verb "distribute" is not accurate.

I don't think it's a big deal either way. At this point in time, I prefer distributed. But happy to revert if you feel strongly against it or if we want to be consistent and consider renaming to distribute/something-else an action that should be take in one go across all repos.